code/obj/item/device/lightbreaker.dm

```diff
@@ -35,6 +35,10 @@
 		return
 
 	proc/activate(mob/user as mob)
+		var/area/a = get_area(user)
+		if(a?.sanctuary)
+			boutput(user, "<span class='alert'>This doesn't seem to work here!</span>")
+			return 0
 		playsound(src.loc, "sound/effects/light_breaker.ogg", 75, 1, 5)
 		for (var/obj/machinery/light/L in view(7, user))
```

Comment:
incorrect implementation.
i can stand 1 tile outside the sanctuary and break the lights inside it.
you want to check the actual lights

Reply:
Changed to lights but turns out the medical room and firing range isn't sanctuary, firing range isn't surprising but medical is an issue. Added a check for the battlecruiser area as well.